docs/pages/blog/2021-q2-update.md

```diff
@@ -0,0 +1,233 @@
+
+```jsx
+<Grid container columns={16}>
+```
+
+We have added an alternative implementaton that uses [CSS grid](https://next.material-ui.com/components/grid/#css-grid-layout):
+
+```jsx
+<Box display="grid" gridTemplateColumns="repeat(12, 1fr)" gap={2}>
+  <Box gridColumn="span 8">
+    <Item>xs=8</Item>
+  </Box>
+  <Box gridColumn="span 4">
+    <Item>xs=4</Item>
+  </Box>
+  <Box gridColumn="span 4">
+    <Item>xs=4</Item>
+  </Box>
+  <Box gridColumn="span 8">
+    <Item>xs=8</Item>
+  </Box>
+</Box>
+```
+
+- 💄 We have improved the accessibility of the Link component:
+
+  <a href="https://next.material-ui.com/components/links/">
+    <img loading="lazy" src="/static/blog/2021-q2-update/link.png" alt="" style="width: 129px; margin-bottom: 16px;" />
+  </a>
+
+  <p class="blog-description">The underline helps to differentiate links in sentences.</p>
+
+### Advanced components
+
+We have primarily focused on the data grid components, fixing a lot of bugs.
+
+#### Date Picker
+
+The date picker is at the border between the core component and advanced components realms.
+
+- 🐛 We have fixed a couple of straightforward problems: visual bugs, use of the `components` prop for customizability, etc.
+- ⚙️ We have continued to focus on addressing technical debt present in the picker components.
+
+#### Data Grid
+
+- 🚨 We have continued to work on breaking changes to reach a stable version.
+  We think that the component is good enough for use in production (and many developers already are using it), but we want to get the public API right before committing to a stable release.
+- ✍️ We have added support for the [cell edit mode](https://material-ui.com/components/data-grid/editing/).
+
+  <img loading="lazy" src="/static/blog/2021-q2-update/cell-edit.gif" alt="" style="width: 842px; margin-bottom: 16px;" />
+
+- 🐛 We have focused on fixing bugs and regressions to ensure early users have a great experience with the component.
+  This was done instead of taking on ambitious new features.
+  We want our users to be able to adopt new features as fast as we build them.
+- 🔘 We have added support for a built-in [single select](/components/data-grid/columns/#column-types) column type:
+
+  <img loading="lazy" src="/static/blog/2021-q2-update/single-select.png" alt="" style="width: 481px; margin-bottom: 16px;" />
+
+  <p class="blog-description">the <a href="https://codesandbox.io/s/material-demo-forked-iuyo5?file=/demo.js">codesandbox</a></p>
+
+  and the **boolean** column type.
+
+- 🚀 We have improved the performance.
+  Client-side sorting and filtering are 2-3x faster on large data sets.
+  We have improved the UX when scrolling.
+  We have added memo logic on the render cell's parent, and worked on its effectiveness to reduce the re-rendering of custom cells.
+  We have identified more opportunities to improve the performance, but we won't prioritize them just yet.
```

Comment:
This seems weird to say. The only thing we're adding here is a frustration to those waiting for performance improvements. Can we justify somehow, positively, why we're not prioritizing them just yet?

Reply:
Ok, I'm changing the wording